Module 1: details popup window

boniface193 opened the pull request:
### popup window that includes the project details in both desktop and mobile

 - When the user clicks (or taps) the button to check project details, the popup with details about the project appears
 - When the user clicks (or taps) the close (X) button, the popup disappears.
 - initial project was refactor to use object to store project details

deboik193 committed:
project single page implemented

deboik193 committed:
popup modal with object items

nxnom committed:
Fix linters

deboik193 committed:
marge conflict solved

emmiiorji commented on .stylelintrc.json:
```diff
@@ -1,11 +1,6 @@
-    "stylelint-scss",
-    "stylelint-csstree-validator"
-  ],
+  "extends": ["stylelint-config-standard"],
+  "plugins": ["stylelint-scss", "stylelint-csstree-validator"],
```
Kindly avoid modifying the linter configurations, they represent coding standards you share with your team.

boniface193 replied:
I understand what you mean (after carefully going through the code I realized the configuration was still the same)

emmiiorji reviewed (changes requested):
Hi @boniface193, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

![Invalid](https://user-images.githubusercontent.com/20554583/209328547-df8bb874-f3dd-4b1b-b50b-f3655b847f82.gif)

You have submitted a project that modifies linters configuration. Please ensure that you use the exact configuration for a [HTML/CSS/JS](https://github.com/microverseinc/linters-config/tree/master/html-css-js) project as given my Microverse. Confirm that the `.eslintrc.json`, `.stylelintrc.json`, `.hintrc` and `.github/workflows/linters.yml` files match the ones in your project.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

Gambit142 reviewed (commented):
Hi @boniface193 @oyhpnayiaw 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Kindly add the following sections to your _**README.md**_ file:

- The **_Key Features_** Section. This section should contain 1 - 3 features that was implemented in this project
- The **_Future Features_** Section. This section should contain 1 - 3 features that can be implemented in the future 
- The `MIT` link in the file leads me to a 404 error page. Kindly fix this and personalize the LICENSE that you will use. 👍 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

boniface193 committed:
README.md changes

###Code review change implemented

 - The Key Features Section.
 - The MIT link in the file leads MIT.md

deboik193 committed:
MIT license added

deboik193 committed:
README file updated

boniface193 commented:
all reame file changes are completed @Gambit142

rustam-tolipov commented on MIT.md:
```diff
@@ -0,0 +1,10 @@
+## Copyright 2021, [IKechukwu]
+
+###### Please delete this line and the next one
+###### APP TYPE can be a webpage/website, a web app, a software and so on
```
[OPTIONAL] I thing this lines of the MIT license is not necessary. You can remove them.

rustam-tolipov commented on style.css:
```diff
@@ -494,6 +613,15 @@ nav ul li img {
 
 /* medium screen resolution */
 @media (min-width: 767px) and (max-width: 1000px) {
+  .grid-container {
```
Your desktop design is excellent. However, there are discrepancies in its mobile design which require attention. Please refer to the attached video:

Solarized dark             |  Solarized Ocean
:-------------------------:|:-------------------------:
![mobile](https://user-images.githubusercontent.com/69011963/209352888-6889a365-7820-42f8-bc70-24f9aa74cd69.gif)   |  ![Template 1-Mobile-Detail](https://user-images.githubusercontent.com/69011963/209354108-05d05523-4265-4c33-bb30-6409f76c3e67.png)

- [x] 1. If I hower your mobile (modal menu) it breaking the design. 
- [x] 2. Your text is a bit longer than the original design. I suggest you to limit it or you can use overflow method. 
- [x] 3. Your stacks (html, css, javaScript) are alse really close to the description.
- [x] 4. As you can see in the video your buttons are also looks different.

Kindly fix all the mentioned changes 😉

rustam-tolipov commented on style.css:
```diff
@@ -690,18 +818,22 @@ nav ul li img {
     padding: 16px 16px 24px 16px;
-    height: min-content;
+
+    /* height: min-content; */
```
Please ensure that any unused commented codes are removed from the file. Thank you.

rustam-tolipov commented on js/projects.js:
```diff
@@ -0,0 +1,91 @@
+            createPopupCard.remove();
+            const getLastChild = Array.from(skipFirstChild);
+            getLastChild[1].lastElementChild.remove();
+            window.reload();
```
- [x] If I open one of your modal menus, close it, and then attempt to open another modal menu, it does not behave as expected. This indicates that your clothing functions require attention. In the browser object model (BOM) there is no reload() functionality. The reload() function can be found in the location object. To access it, call window, then location and finally reload(). The full form is window.location.reload(), which should solve your issue when implemented correctly.

rustam-tolipov reviewed (commented):
Hello @boniface193,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
_Your project is almost complete!_
_No Linters errors_

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

**You are totally free to remove this lines in your Readme:**

_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._

_Check the comments under the review._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
